### DISCUSSION TITLE
makemigrations giving errors

### DISCUSSION THREAD

**Discussion by Shea766** [netbox-community/netbox#9004] [Q&A] [answered]
I'm not able to run migrate or makemigrations:

python3 manage.py migrate
Operations to perform:
  Apply all migrations: admin, auth, circuits, contenttypes, dcim, django_rq, extras, ipam, myfun_plugin, sessions, social_django, taggit, tenancy, users, virtualization, wireless
Running migrations:
  No migrations to apply.
  Your models in app(s): 'myfun_plugin' have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.

Ok, so I run 'manage.py makemigrations':

python3 manage.py makemigrations
CommandError: This command is available for development purposes only. It will
NOT resolve any issues with missing or unapplied migrations. For assistance,
please post to the NetBox discussion forum on GitHub:
    https://github.com/netbox-community/netbox/discussions

So now I'm posting to discussions.
What am I doing wrong?

**Comment by jeremystretch** [comment, marked as answer]
You need to set the [`DEVELOPER`](https://netbox.readthedocs.io/en/stable/configuration/optional-settings/#developer) config parameter to True to enable the command. This is a safeguard to prevent people from mistakenly creating erroneous migrations and corrupting their database.

**Reply by Shea766** [reply]
thanks
